packages/e2e-tests/src/plugins/web-stories-taxonomies.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Plugin Name: Web Stories Test Plugin: Taxonomies
+ * Plugin URI:  https://github.com/google/web-stories-wp
+ * Description: Test plugin that adds some custom taxonomies.
+ * Author:      Google
+ * Author URI:  https://opensource.google.com/
+ */
+
+namespace Google\Web_Stories\E2E_Tests;
+
+function add_taxonomies() {
+	register_taxonomy(
+		'story-color',
+		'web-story',
+		[
+			'description'  => 'Story Colors',
+			'show_in_rest' => true,
+			'show_ui'      => true,
+			'rest_base'    => 'story-colors',
```

Comment:
I think this needs to be `web_story_colors`
```suggestion
			'rest_base'    => 'web_story_colors',
```

Reply:
The name of `rest_base` doesn't matter. For some third-party custom Acme taxonomy not in our control it might be `acme-super-tax1234`. Doesn't matter.

Reply:
I don't see anywhere in the FE code where we necessitate a `web_story_` prefix. The only things I see that allow a taxonomy to show up are the need for these conditions to be true:
https://github.com/google/web-stories-wp/blob/35a65a98f5059ad3454b53812473c774fb6d5a21/packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js#L43-L48

Reply:
ah good catch, yeah as long as those capabilities are there then it should wrok

Reply:
Is this what we're thinking? https://github.com/google/web-stories-wp/pull/9220

Reply:
Yes.